topic-operator/src/main/java/io/strimzi/operator/topic/K8sTopicWatcher.java

```diff
@@ -74,6 +81,10 @@ public void eventReceived(Action action, KafkaTopic kafkaTopic) {
         }
     }
 
+    protected boolean hasPauseReconciliationAnnotation(CustomResource resource) {
+        return Annotations.booleanAnnotation(resource, ANNO_STRIMZI_IO_PAUSE_RECONCILIATION, false);
+    }
```

Comment:
Can we have this only once? Maybe in some Util class or in the Annotations class? Also, same comment as before about the name.